### ISSUE TITLE
All my download queue completely disappeared

### ISSUE THREAD

**Issue by alejandroiglesias** [arranger1044/iSoul#47]
Hi, all of a sudden, after restarting the app, all my download queue completely disappeared. This might be related to running out of HD space. Is there any log that can be checked to see what happened? Is there any way of restoring my queue?
Thanks in advance.

**Comment by arranger1044** [comment]
Yes there is a log file you can visualize by opening the Console in iSoul.
However I doubt there is a way to restore your queue, sorry.
